src/components/experiments/wizard/Metrics.tsx

```diff
@@ -126,7 +126,7 @@ const createMetricAssignment = (metric: Metric) => {
   return {
     metricId: metric.metricId,
     attributionWindowSeconds: '',
-    isPrimary: false,
+    isPrimary: true,
     changeExpected: false,
```

Comment:
Judging by the PR description ("Makes `changeExpected` true by default"), did you mean to make this change instead?

```suggestion
    isPrimary: false,
    changeExpected: true,
```

Reply:
🤦